test/unit/Input.test.js

```diff
@@ -118,15 +118,15 @@ describe('Input with (default) type=text', () => {
             input = mount(<Input value="initial value" />);
         });
         test('Input updates value', () => {
-            expect(input.find('input').getNode().value).toEqual(
+            expect(input.find('input').instance().value).toEqual(
```

Comment:
Similar question here, what is the difference between `getNode()` and `instance()`? An API change in `enzyme`?

Reply:
Exactly. Enzyme got updated, and a bunch of the methods I used before didn't work anymore after upgrading.